packages/patternfly-3/patternfly-react/src/components/DateTimePicker/TimeComponents/PickTimeClock.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from '../../../common/helpers';
+
+class PickTimeClock extends React.Component {
+  state = {
+    ampm: this.props.time.getHours() >= 12 ? 'PM' : 'AM'
+  };
+  static getDerivedStateFromProps(nextProps, prevState) {
+    return {
+      ampm: nextProps.time.getHours() >= 12 ? 'PM' : 'AM'
+    };
+  }
+  setTime = (type, amount) => {
+    const { time } = this.props;
+    if (type === 'H') {
```

Comment:
can you create constants for these? 
you can call them HOUR, MINUTE etc - it would be more readable.

Reply:
Added consts HOUR, MINUTE, AM ,PM. thanks!